FinniversKit/Sources/Components/Consent/BannerTransparency/BannerTransparencyViewModel.swift

```diff
@@ -28,6 +30,13 @@ extension BannerTransparencyViewModel {
     }
 }
 
+// MARK: - LogoType Enum
+
+public enum LogoType {
+    case finn
+    case tori
```

Comment:
It is a matter of taste but I would add a `logoImage` var to the enum and then move this

```
            switch model.logoType {
             case .finn:
                 logoImageView.image = UIImage(named: .finnLogo)
             case .tori:
                 logoImageView.image = UIImage(named: .toriLogo)
             }
```

inside of that. 

So something like: 

```
var logoImage: UIImage {
    switch self {
         case .finn:
             UIImage(named: .finnLogo)
         case .tori:
             UIImage(named: .toriLogo)
    }
}
```

And then you could use that like so in your `BannerTransparencyView`

`logoImageView.image = model.logoType.logoImage`

(OK lots if `image`s in there, maybe clear that up 😄 )

But again, your taste may vary 😄

Reply:
Simplified it based on Björns suggestions so have removed this now 😅  But thank you for your input!